src/main/java/com/monov/cloud/gateway/service/StudentGatewayService.java

```diff
@@ -0,0 +1,51 @@
+package com.monov.cloud.gateway.service;
+
+import com.monov.cloud.gateway.dto.Course;
+import com.monov.cloud.gateway.dto.Student;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+@Service
+public class StudentGatewayService {
+
+    public static final String STUDENT_SERVICE_URL = "http://localhost:9001/students";
```

Comment:
Move this to .properties / .yaml

Reply:
Fixes will be applied in DTOs branch's PR